winterfell/webapp/src/styles/form.styl

```diff
@@ -30,19 +30,19 @@
     left: 0;
     right: 0;
 
-.form-wrapper
+#vfi-form-view
   margin-left: 10%
   margin-right: 10%
 
-.form-submit-button-group
-  margin-top: 3vh
+  & .form-submit-button-group
```

Comment:
when you move all these classes under the parent id,
you can remove confusion and mixups with all `.form-title`, `.form-submit-button-group`, `.form-name`, etc.
You can know for sure that all of those classes are used in this page and only this page.